runtime/manifest.js

```diff
@@ -30,6 +30,20 @@ class ManifestError extends Error {
   }
 }
 
+class StorageStub {
+  constructor(type, id, name, storageKey, storageProviderFactory) {
+    this.type = type;
+    this.id = id;
+    this.name = name;
+    this.storageKey = storageKey;
+    this.storageProviderFactory;
+  }
+
+  inflate() {
+    return this.storageProviderFactory.connect(this.id, this.type, this.storageKey);
```

Comment:
I filed https://github.com/PolymerLabs/arcs/issues/1442 to track adding unit tests for this and the logic above which is fine to do separately.